sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesOperation.cs

```diff
@@ -331,7 +331,7 @@ async ValueTask<OperationState<AsyncPageable<AnalyzeHealthcareEntitiesResultColl
                 AnalyzeHealthcareEntitiesResultCollection value = Transforms.ConvertToAnalyzeHealthcareEntitiesResultCollection(response.Value.Results, _idToIndexMap);
                 _firstPage = Page.FromValues(new List<AnalyzeHealthcareEntitiesResultCollection>() { value }, nextLink, rawResponse);
 
-                return OperationState<AsyncPageable<AnalyzeHealthcareEntitiesResultCollection>>.Success(rawResponse, CreateOperationValueAsync());
+                return OperationState<AsyncPageable<AnalyzeHealthcareEntitiesResultCollection>>.Success(rawResponse, CreateOperationValueAsync(cancellationToken));
```

Comment:
I don't think it's correct to use this `cancellationToken` here. This one would be for the `UpdateStateAsync` operation we don't want to capture it inside an IAsyncEnumerable.

Reply:
removed